Bigtable/src/Table.php

```diff
@@ -0,0 +1,351 @@
+     * @codingStandardsIgnoreEnd
+     *
+     * @todo add options for ColumnFamily
+     * @param array $options [optional] Configuration options.
+     * @return array Table information
+     */
+    public function create(array $options = [])
+    {
+        $instanceName = TableAdminClient::instanceName($this->projectId, $this->instanceId);
+        $this->info = $this->connection->createTable([
+            'parent' => $instanceName,
+            'tableId' => $this->id
+        ] + $options);
+
+        return $this->info;
+    }
+
+    /**
+     * Delete the current table.
+     *
+     * Example:
+     * ```
+     * $table->delete();
+     * ```
+     *
+     * @param array $options [optional] Configuration options.
+     *
+     * @return void
+     */
+    public function delete(array $options = [])
+    {
+        return $this->connection->deleteTable([
+            'name' => $this->name
+        ] + $options);
+    }
+
+    /**
+     * Add column families to the current table.
+     *
+     * Example:
+     * ```
+     * $tableInfo = $table->addColumnFamilies([
+     *     ['id' => 'cf1', 'gcRule'=>[]],
+     *     ['id' => 'cf2', 'gcRule'=>[]],
+     *     ['id' => 'cf3', 'gcRule'=>[]]
+     * ]);
+     * ```
+     * @todo add GCRule options
+     * @param array[] $columnFamilies List of ColumnFamilies, each entry in the list
+     *        is array with key `id` for id of ColumnFamily
+     * @param array $options [optional] Configuration options.
+     * @return array Table information
+     */
+    public function addColumnFamilies(array $columnFamilies, array $options = [])
+    {
+        $modifications = [];
+        foreach ($columnFamilies as $columnFamily) {
+            $modifications[] = ['id' => $columnFamily['id'], 'create' => $columnFamily['gcRule']];
+        }
+        $this->info = $this->connection->modifyColumnFamilies([
+            'name' => $this->name,
+            'modifications' => $modifications
+        ] + $options);
+
+        return $this->info;
+    }
+
+    /**
+     * Drop column families to the current table.
+     *
+     * Example:
+     * ```
+     * $tableInfo = $table->dropColumnFamilies(['cf1', 'cf2', 'cf3']);
+     * ```
+     *
+     * @param array $columnFamilies List of ColumnFamilies to be dropped
+     * @param array $options [optional] Configuration options.
+     * @return array Table information
+     */
+    public function dropColumnFamilies(array $columnFamilies, array $options = [])
+    {
+        $modifications = [];
+        foreach ($columnFamilies as $columnFamily) {
+            $modifications[] = [ 'id' => $columnFamily, 'drop' => true];
+        }
+        $this->info = $this->connection->modifyColumnFamilies([
+            'name' => $this->name,
+            'modifications' => $modifications
+        ] + $options);
+
+        return $this->info;
+    }
+
+    /**
+     * Update column families to the current table.
```

Comment:
`s/to/on`

Reply:
@ajaaym could you fix this last thing real quick? after that good to merge.

Reply:
done. @jdpedrie  Thanks for pointing this out.